source/backend/api/Areas/Reports/Mapping/Lease/LeaseMap.cs

```diff
@@ -36,7 +36,7 @@ private static void MapLease((Entity.PimsLeasePeriod period, Entity.PimsLease le
             dest.InspectionNotes = src.lease.InspectionNotes;
             dest.InspectionDate = src.lease.InspectionDate?.FilterSqlMinDate();
             dest.LeaseNotes = src.lease.LeaseNotes;
-            dest.IsExpired = (src.lease.GetExpiryDate() < DateTime.Now).BoolToYesNo();
+            dest.IsExpired = (src.expiryDate < DateTime.Now).BoolToYesNo();
```

Comment:
Do we need to include renewals in this calculation?

Reply:
Not 100%. I dont think this field is actually used in the frontend. But I will clean it up as tech debt